samples/drivers/ipm/ipm_esp32/CMakeLists.txt

```diff
@@ -4,12 +4,12 @@ cmake_minimum_required(VERSION 3.20.0)
 
 set(REMOTE_ZEPHYR_DIR ${CMAKE_CURRENT_BINARY_DIR}/ipm_esp32_appcpu-prefix/src/ipm_esp32_appcpu-build/zephyr)
 
-if("${BOARD}" STREQUAL "esp32_devkitc_wrover")
-  set(BOARD_REMOTE "esp32_devkitc_wrover_appcpu")
-elseif("${BOARD}" STREQUAL "esp32_devkitc_wroom")
-  set(BOARD_REMOTE "esp32_devkitc_wroom_appcpu")
-elseif("${BOARD}" STREQUAL "esp32s3_devkitm")
-  set(BOARD_REMOTE "esp32s3_devkitm_appcpu")
+if("${BOARD}" STREQUAL "esp32_devkitc_wrover/esp32/procpu")
+  set(BOARD_REMOTE "esp32_devkitc_wrover/esp32/appcpu")
+elseif("${BOARD}" STREQUAL "esp32_devkitc_wroom/esp32/procpu")
+  set(BOARD_REMOTE "esp32_devkitc_wroom/esp32/appcpu")
+elseif("${BOARD}" STREQUAL "esp32s3_devkitm/esp32s3/procpu")
```

Comment:
This sample should be updated to use sysbuild because it is wrong but that is a future task, as for here the if() names don't look to have been updated and are still using Xcpu

Reply:
I have not yet updated cpu prefix. Anyway, I'll update the sample code to use sysbuild in a new PR.